src/main/java/org/greenbuttonalliance/gbaresourceserver/usage/model/ReadingQuality.java

```diff
@@ -29,6 +29,7 @@
 import lombok.Getter;
 import lombok.Setter;
 import lombok.experimental.Accessors;
+import org.greenbuttonalliance.gbaresourceserver.usage.model.enums.QualityOfReading;
```

Comment:
I made a new package for enums and moved the existing `QualityOfReading.java` file there, so it now needs to be explicitly imported here